### ISSUE TITLE
Overfitting

### ISSUE THREAD

**Issue by sr5434** [Zhenye-Na/self-driving-vehicles-sim-with-ml#7]
Hello! I reimplemented this repo in Keras, and trained it on the default dataset. However, I notice that the validation loss increases after the first epoch, which is a crazy amount of overfitting. After spiking, the loss doesn’t seem to come down. This screenshot is without dropout, but with dropout the overfitting is even worse!
![IMG_1242](https://github.com/Zhenye-Na/e2e-learning-self-driving-cars/assets/118690585/b75a7c83-39d5-4c8c-be89-d5bb0fad6244)
Thank you for any help you can provide.

**Comment by sr5434** [comment]
Nevermind, I was using the wrong LR decay(cosine instead of piecewise constant)
